radsat-sk/operation/subsystems/RObc.c

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file RObc.h
+ * @date April 2, 2022
+ * @author Matthew Buglass (mab839)
+ */
+
+#include <RObc.h>
+#include <RCommon.h>
+#include <hal/errors.h>
+#include <hal/Timing/WatchDogTimer.h>
+#include <hal/Timing/RTC.h>
+
+/***************************************************************************************************
+                                             PUBLIC API
+***************************************************************************************************/
+
+/**
+ * Retrieve Temperature telemetry from OBC
+ *
+ * @return	Error code; 0 for success, otherwise see hal/errors.h.
+ */
+int obcTelemetry(obcTelemetry_t* obcTelemetryBuffer){
+	float temperature = 0;
+	int error = RTC_getTemperature(&temperature);
+
+	if (error != SUCCESS) {
+		return error;
+	}
+
+	obcTelemetryBuffer->temperature = temperature;
+	return SUCCESS;
+
+}
+
+/**
+ * Pet the watchdog on the OBC
+ *
+ * @return	N/A
+ */
+void obcPetWatchdogs(void){
+	WDT_forceKick();
+}
+
```

Comment:
This function probably isn't necessary; we already have `WDT_startWatchdogKickTask` provided by the HAL which is supposed to start a FreeRTOS task to do all of the kicking for us. At the very least, this function should mention the existence of that HAL function.